config/environments/development.rb

```diff
@@ -22,8 +22,9 @@
   if Rails.root.join("tmp/caching-dev.txt").exist?
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
+    config.action_controller.page_cache_directory = Rails.public_path
```

Comment:
Your PR does not explain why you are making this change, which affects all users not just docker users.